_orders/querying.md

```diff
@@ -0,0 +1,498 @@
+---
+title: Querying
+layout: default
+nav_order: 1
+---
+Beta
+{: .label .label-yellow }
+
+# Orders Query Implementation Guide
+
+## Overview
+Your system needs to query your orders the point services platform.
+
+## Obtaining Authentication Tokens
+
+Before you can test your query, you must obtain an access token for the pointservices platform.
+
+### Step 1: Authenticate and Receive Tokens
+
+Use the following command to authenticate with your credentials and receive your token. Replace `customerUsername` and `customerPassword` with your actual username and password.
+
+```bash
+curl -X POST https://api.pointservices.com/user-management-services-ws/oauth2/002/signInWithPassword \
+-H "Content-Type: application/json" \
+-d '{
+  "username": "customerUsername",
+  "password": "customerPassword"
+}'
+```
+
+The response will be a json structure that includes:
+- `access_token`: Used for authenticated requests to the webhook testing endpoint.
+- `refresh_token`: Used to obtain a new access token once the current one expires.
+- `expires_in`: The time in seconds until the access token expires.
+
+Read the response to obtain the new `access_token`, `expires_in` and `refresh_token` values. The `expires_in` value will in most cases be 3600 seconds (1 hour) from the time the new access token was issued.
+
+Example response:
+
+```json
+{
+  "access_token": "eyJraWQiOiIxMjM0NSIsInR5cCI6IkpXVCIsImFsZyI6IlJTMjU2In0.eyJhdWQiOiJ0ZXN0LXRlbmFudCIsInN1YiI6InhwcHN8MTIyMyIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJ1c2VyX2lkIjoiMTIyMyIsImF1dGhfdGltZSI6MTcwODcxNTYwMiwiaXNzIjoiaHR0cHM6XC9cL3NlY3VyZXRva2VuLmdvb2dsZS5jb21cL3Rlc3QtdGVuYW50IiwiZXhwIjoxNzA4NzE5LCJpYXQiOjE3MDg3MTUsImVtYWlsIjoidGVzdHVzZXJAdGVuYW50LmNvbSJ9.dGVzdA==",
+  "refresh_token": "AMf-vBxITbnfBBMJpPXMvop1qa1cMaUuqAKX_y1hYsmcgVtzhx7Al_9mWD",
+  "expires_in": 3600
+}
+```
+### Step 2: Send a request to list the first 100 orders under your account
+
+Incorporate the access token in the `Authorization` header and send a test request as follows:
+
+```bash
+curl -X POST https://api.pointservices.com/riskinsight-services-ws/resources/v1/orders/query 
+-H "Authorization: Bearer your_access_token_here" 
+-H "Content-Type: application/json" 
+-d ''
```

Comment:
write an actual json example instead of leaving it blank

Reply:
actually, put a slightly complicated example with 2 items to filter -- that way there's an example at least of how it's written

Reply:
This is the query for when you don't know what you're querying for.